hack/boilerplate/test/fail.go

```diff
@@ -1,7 +1,7 @@
 /*
 Copyright 2014 The Kubernetes Authors.
 
-fail
+Fail
```

Comment:
That would be something we have to exclude if we want to keep the capital letter linting

Reply:
Yeah, might be good to just add all `../test/*` paths to the ignore list.

Reply:
Should be fine now, I dropped the capitalization.